app/src/main/java/de/westnordost/streetcomplete/data/user/statistics/StatisticsController.kt

```diff
@@ -178,11 +178,11 @@ class StatisticsController(
         val today = systemTimeNow().toLocalDate()
         val lastUpdateDate = Instant.fromEpochMilliseconds(lastUpdate).toLocalDate()
         lastUpdate = nowAsEpochMilliseconds()
+        activeDatesDao.addToday()
```

Comment:
Why was this moved? It was not explained in the commit message.

Reply:
It was an unrelated bugfix, committed to master now